create_post.php

```diff
@@ -0,0 +1,60 @@
+<?php
+	include('include/databaseFunctions.php');
+
+	$type=$_REQUEST['type'];
+	echo"
+		<html>
+			<head>
+				<title>Create Your Own Post</title>
+				<link rel='stylesheet' href='style.css'>
+			</head>
+			<body>
+				<h1>Create Your Own";
+				if($type == 1){
+					echo" Picture ";
+				}else{
+					echo" Blog ";
+				}
+
+	echo							"Post</h1>
+				<form method='post'>";
+				if($type== 1){
+					echo"<p class='required'>Photographer*:</p><input type='text' name='photographer'><br><br>
+						 <p class='required'>Title*:</p><input type='text' name='title'><br><br>
+						 Post Body (optional):<br><textarea name='body' rows='5' cols='40'></textarea><br><br>
+						 <p class='required'>Photo Path*:</p><input type='text' name='link'><br><br>
+						 Flavortext:<br><input type='text' name='flavor'><br><br>";
+				}else{
+					echo"Author:<br><input type='text' name='author'><br><br>
+						 <p class='required'>Title*:</p><input type='text' name='title'><br><br>
+						 <p class='required'>Post Body*:</p><textarea name='body' rows='5' cols='40'></textarea><br><br>";
+				}
+
+	echo"
+					<input type='submit'>
+				</form>";
+		if($type == 1){
+			if(isset($_REQUEST['photographer'])&&isset($_REQUEST['title'])&&isset($_REQUEST['link'])){
```

Comment:
This is really good for something you figured out yourself, but I'd guess you have some ideas on how to improve it based on today's forms session. Why don't you take another crack at it and let me know when that's ready for review.